core/genesis.go

```diff
@@ -382,7 +382,7 @@ func DefaultGoerliGenesisBlock() *Genesis {
 
 func DefaultYoloV1GenesisBlock() *Genesis {
```

Comment:
Nit, pls rename to YoloV2. I know it didn't change, but seems nicer to get rid of all v1 mentions in the code. Also pls udate yoloV1AllocData